### PR TITLE
Add print button for tooltip/legend popup

### PR DESCRIPTION
**Pull request by pakb**
[Test link](https://web-mapviewer.dev.bgdi.ch/feature_BGDIINF_SB-1701/index.html)

### CODE REVIEW

**Review by ltshb — COMMENTED** [review]
From the code as far as I can tell it is good. However I tested it with the Public transport stops layer and the time table of the tooltips doesn't get printed: 
image
Result to 
image

**Comment by pakb** [comment]
ho, that's a case I didn't think about... that's no image but an `iframe`... I should also wait for them to be finished loading before printing.
Funny thing is, I also tried a bus stop and it worked...

**Comment by davidoesch** [comment]
the one under https://mf-geoadmin3.int.bgdi.ch/?ch.swisstopo.lubis-luftbilder_schwarzweiss=19832100010218&time=None&lang=en&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.lubis-luftbilder_schwarzweiss,ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_opacity=1,1,1,1,0.8&layers_visibility=true,false,false,false,false&layers_timestamp=99991231,18641231,,,&E=2711534.65&N=1210622.10&zoom=6

-> print window closes.... (image inside)

**Comment by davidoesch** [comment]
same fore my favorite edge case https://web-mapviewer.dev.bgdi.ch/feature_BGDIINF_SB-1701/index.html#/map?lat=47.032444&lon=8.893587&z=12.278&layers=ch.bfe.windenergie-geschwindigkeit_h100

**Comment by pakb** [comment]
😆 this one is no edge case, it's a nightmare

for `ch.swisstopo.lubis-luftbilder_schwarzweiss` it seems to work for me with my latest fix (wait for it to be published on the test link, should be about right now...)

I'll dig into our beloved Windrose...

**Comment by pakb** [comment]
well, if you clean your cache, and retry the link you've sent me, my latest fix including `iframe` into the mix seems to have done it...

File: src/utils/print.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    document.write('</body></html>')
+    document.close()
+
+    const printAndClose = () => {
+        hiddenWindowWithContent.print()
+        hiddenWindowWithContent.close()
+    }
+    // checking if there are images or iframe in the content
+    let pendingImageOriFrameLoading = 0
+    const images = jQuery(document).find('img,iframe')
```
Comment (ltshb) [inline src/utils/print.js:31]: Just wondering if we know cover all cases with img and iframe ?

**Review by ltshb — APPROVED** [review]

File: src/utils/print.js (hunk 1 above)
Comment (pakb) [reply src/utils/print.js:31]: I don't see another use case with our HTML popup. Usually it's a simple HTML table with a couple links at the end. Sometimes there's an image, and in some advanced cases there's an iFrame (`ch.bav.haltestellen-oev` and `ch.bfe.windenergie-geschwindigkeit_h50` are prime examples)

@rebert @ltclm Do you know any other cases of HTML popup that have content that needs loading other than `iFrame` and `img`?

Comment (ltclm) [reply src/utils/print.js:31]: here are some more layers which are using javascript in htmlpopup (or extended tooltip):
```
cadastralwebmap.include.mako  --> ch.kantone.cadastralwebmap-farbe
cadastralwebmap_opendata.mako --> ch.swisstopo-vd.amtliche-vermessung
kernkraftwerke.mako --> ch.bfe.kernkraftwerke
kgs.mako --> ch.babs.kulturgueter
klaeranlagen.mako --> ch.bafu.gewaesserschutz-klaeranlagen_anteilq347, ch.bafu.gewaesserschutz-klaeranlagen_reinigungstyp, ch.bafu.gewaesserschutz-klaeranlagen_ausbaugroesse
lubis_bildstreifen.mako --> ch.swisstopo.lubis-bildstreifen
lubis.mako --> ch.swisstopo.lubis-luftbilder_farbe, ch.swisstopo.lubis-luftbilder_farbe, ch.swisstopo.lubis-luftbilder_schwarzweiss, ch.swisstopo.lubis-luftbilder-dritte-firmen, ch.swisstopo.lubis-luftbilder-dritte-kantone
lubis_schraegaufnahmen.mako --> ch.swisstopo.lubis-luftbilder_schraegaufnahmen
lubis_terra.mako --> ch.swisstopo.lubis-terrestrische_aufnahmen
luftfahrthindernisse.mako --> ch.bazl.luftfahrthindernis
oev_haltestellen.mako --> ch.bav.haltestellen-oev 
statistikwasserkraftanlagen.mako --> ch.bfe.statistik-wasserkraftanlagen
verkehrszaehlstellen.mako --> ch.bfe.statistik-wasserkraftanlagen
windatlas50.mako --> ch.bfe.windenergie-geschwindigkeit*
windenergieanlagen_facility.mako --> ch.bfe.windenergieanlagen
windenergieanlagen_turbine.mako --> ch.bfe.windenergieanlagen
```
since with most of these layers are loading javascript and other stuff in the extended tooltips only, there are some special layers with javascript in the default tooltip / htmlpopup:
* ch.bfe.windenergie-geschwindigkeit*
* ch.bav.haltestellen-oev

i dont think that there are more exceptions than these but i might be wrong.
